spec/controllers/api/v1/subjects_controller_spec.rb

```diff
@@ -309,23 +309,33 @@
     let(:test_attr) { :metadata }
     let(:test_attr_value) do
       {
-       "interesting_data" => "Tested Collection",
-       "an_interesting_array" => ["1", "2", "asdf", "99.99"]
+        "interesting_data" => "Tested Collection",
+        "an_interesting_array" => ["1", "2", "asdf", "99.99"]
       }
     end
     let(:update_params) do
       {
-       subjects: {
-                  metadata: {
-                             interesting_data: "Tested Collection",
-                             an_interesting_array: ["1", "2", "asdf", "99.99"]
-                            },
-                  locations: [ "image/jpeg" ]
-                 }
+        subjects: {
+          metadata: {
+            interesting_data: "Tested Collection",
+            an_interesting_array: ["1", "2", "asdf", "99.99"]
+          },
+          locations: [ "image/jpeg" ]
+        }
       }
     end
 
     it_behaves_like "is updatable"
+
+
+    it 'should create externally linked media resources' do
+      default_request user_id: authorized_user.id, scopes: scopes
+      update_params[:subjects].merge!(locations: [{"image/jpeg" => "http://example.com/1.jpg"},
+        {"image/jpeg" => "http://example.com/2.jpg"}])
+      put :update, update_params.merge(id: resource.id)
+      locations = Subject.find(created_instance_id("subjects")).locations
+      expect(locations.map(&:external_link)).to all( be true )
```

Comment:
for coverage i think we should we test the explicit addition of the supplied URL to the `locations.src` value?

``` ruby
external_locs = [{"image/jpeg" => "http://example.com/1.jpg"}, {"image/jpeg" => "http://example.com/2.jpg"}]
external_src_urls = external_locs.map { |loc| loc.to_a.flatten[1] }
#...
aggregate_failures "external srcs" do
  expect(locations.map(&:external_link)).to all( be true )
  expect(locations.map(&:src)).to match_array(external_src_urls)
end
```